tests/tmp/.gitignore

```diff
@@ -1,4 +0,0 @@
-# Ignore everything in this directory
```

Comment:
Since `git` won't retain empty directories, I previously used this trick to ensure that a fresh `git clone` has a mostly empty directory for other use cases.  The `tmp` directory is no longer needed, so removing it.

Reply:
Neat trick.